website/vercel.json

```diff
@@ -1201,6 +1201,36 @@
     {
       "source": "/getting-started/tutorials/customer-support-tool/application-settings",
       "destination": "/getting-started/tutorials/start-building"
+    },
+    {
+      "source": "/widget-reference/file-picker",
+      "destination": "/reference/widgets/filepicker",
+      "permanent": true
+    },
+    {
+      "source": "/widget-reference/map",
+      "destination": "/reference/widgets/maps",
+      "permanent": true
+    },
+    {
+      "source": "/widget-reference/dropdown",
+      "destination": "/reference/widgets/select",
+      "permanent": true
+    },
+    {
+      "source": "/reference/datasources/querying-mongodb/whitelist-mongodb-cloud?q=error",
+      "destination": " /connect-data/reference/querying-mongodb/whitelist-mongodb-cloud",
+      "permanent": true
+    },
+    {
+      "source": "/write-code/reference/core-concepts/writing-code/ext-libraries",
+      "destination": "/core-concepts/writing-code/ext-libraries",
+      "permanent": true
+    },
+    {
+      "source": "/datasource-reference/querying-mongodb/whitelist-mongodb-cloud",
```

Comment:
Not needed. As we already have a rule:
```
      "source": "/datasource-reference/querying-mongodb/whitelist-mongodb-cloud",
      "destination": "/reference/datasources/querying-mongodb/whitelist-mongodb-cloud"
```
The above rule will handle the redirect and the whitelist rule will take users to the whitelist external link.

Reply:
removed